cmd/notation/policy/import.go

```diff
@@ -0,0 +1,110 @@
+package policy
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation/internal/cmd"
+	"github.com/spf13/cobra"
+)
+
+type importOpts struct {
+	cmd.LoggingFlagOpts
```

Comment:
Adding this because `-d` and `-v` are [in the spec](https://github.com/notaryproject/notation/blob/main/specs/commandline/policy.md#notation-policy-show)

Reply:
if we are not logging anything, and there's no http round trips, we could just drop these flags from the spec.